### PR TITLE
feat(SentryApps): Create db migration to add new integration feature choices

### PR DESCRIPTION
**Pull request by NisanthanNanthakumar**
## Objective
Certain Sentry Apps still have the default integration feature of `api` that doesn’t actually explain what the integration does.

**Amixr** should have the category `incident-management` and **ClickUp** should be `project-management`. Spoke with Phillip and decided to keep the text the same.

We will need to run the following SQL scripts in the production environment.

## SQL Scripts
_Script to update Amixr_
```
UPDATE "public"."sentry_integrationfeature" SET "feature"=5 WHERE "sentry_app_id"=(SELECT id from "public"."sentry_sentryapp" WHERE "slug"='amixr') RETURNING "id", "user_description", "feature", "date_added", "sentry_app_id";
```
_Script to update Clickup_
```
UPDATE "public"."sentry_integrationfeature" SET "feature"=4 WHERE "sentry_app_id"=(SELECT id from "public"."sentry_sentryapp" WHERE "slug"='clickup') RETURNING "id", "user_description", "feature", "date_added", "sentry_app_id";
```

## Test Plan
Ran the SQL scripts and DB migration in the local environment and everything looked good.

## UI
Screen Shot 2020-03-30 at 12 43 21 PM
Screen Shot 2020-03-30 at 12 43 10 PM

### CODE REVIEW

**Review by wedamija — COMMENTED** [review]
Could you included the generated DDL? Instructions here: https://www.notion.so/sentry/Django-migrations-103e5994029d4345b5b6f5f6f9cac9c7#e043d9ff341749889e7777640de56ea3

This should cause no ddl changes, but worth checking.

Usually when we want to make a data migration we also use Django migrations. That way the migration will be applied to single tenant and on-premise installs as well. Check out migration 0053 for an example of how to do so. I recommend you generate a separate migration and do your data migration there: `sentry django makemigrations sentry --empty`

File: src/sentry/models/integrationfeature.py [hunk 1]
```diff
@@ -11,6 +11,8 @@ class Feature(object):
     ISSUE_LINK = 1
     STACKTRACE_LINK = 2
     EVENT_HOOKS = 3
+    PROJECT_MANAGEMENT = 4
+    INCIDENT_MANAGEMENT = 5
```
Comment (scefali) [inline src/sentry/models/integrationfeature.py]: @NisanthanNanthakumar So I asked @jonesphillip to come up with updated categories for our integrations. He wants to add one for "feature flag". See the card: https://getsentry.atlassian.net/browse/API-884

File: src/sentry/models/integrationfeature.py [hunk 2]
```diff
@@ -31,10 +35,14 @@ def as_str(cls, feature):
...
             return "integrations-event-hooks"
+        elif feature == cls.PROJECT_MANAGEMENT:
+            return "integrations-project-managment"
+        elif feature == cls.INCIDENT_MANAGEMENT:
+            return "integrations-incident-management"
 
     @classmethod
     def description(cls, feature, name):
-        if feature == cls.API:
+        if feature in [cls.API, cls.PROJECT_MANAGEMENT, cls.INCIDENT_MANAGEMENT]:
```
Comment (scefali) [inline src/sentry/models/integrationfeature.py]: @jonesphillip @NisanthanNanthakumar I think we should definitely have new text for these new categories. I'm curious why the decision was not to do that.

**Comment by mattrobenolt** [comment]
> Usually when we want to make a data migration we also use Django migrations.

I’d we do this, out it into getsentry since what what we’re doing is highly specific to SaaS.

**Comment by NisanthanNanthakumar** [comment]
> Could you included the generated DDL? Instructions here: https://www.notion.so/sentry/Django-migrations-103e5994029d4345b5b6f5f6f9cac9c7#e043d9ff341749889e7777640de56ea3
> 
> This should cause no ddl changes, but worth checking.
@wedamija not sure what the expected output should be, but when I ran the command, I got:
Screen Shot 2020-03-30 at 2 00 05 PM

**Comment by wedamija** [comment]
> > Could you included the generated DDL? Instructions here: https://www.notion.so/sentry/Django-migrations-103e5994029d4345b5b6f5f6f9cac9c7#e043d9ff341749889e7777640de56ea3
> > This should cause no ddl changes, but worth checking.
> > @wedamija not sure what the expected output should be, but when I ran the command, I got:
> 
> Screen Shot 2020-03-30 at 2 00 05 PM

Thanks, that looks good to me.

File: src/sentry/models/integrationfeature.py [hunk 3]
```diff
@@ -31,10 +37,16 @@ def as_str(cls, feature):
...
+            return "integrations-project-managment"
+        elif feature == cls.INCIDENT_MANAGEMENT:
+            return "integrations-incident-management"
+        elif feature == cls.FEATURE_FLAG:
+            return "integrations-feature-flag"
 
     @classmethod
     def description(cls, feature, name):
-        if feature == cls.API:
+        if feature in [cls.API, cls.PROJECT_MANAGEMENT, cls.INCIDENT_MANAGEMENT, cls.FEATURE_FLAG]:
```
Comment (scefali) [inline src/sentry/models/integrationfeature.py]: @NisanthanNanthakumar This fine for now but note that I will update this text in an upcoming PR so they aren't all the same

**Review by scefali — APPROVED** [review]

**Review by wedamija — APPROVED** [review]
This looks good to me. I also looked at how many rows those update scripts change, and it look like they just update one row each, so should be fine for ops to run.

**Pull request merged**